server/Entities/RepositoryContext.cs

```diff
@@ -2,81 +2,97 @@
 using Entities.Models.Enums;
 using Microsoft.EntityFrameworkCore;
 using System;
-using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace Entities
 {
-    public class RepositoryContext : DbContext
+  public interface IRepositoryContext
+  {
+    DbSet<User> Users { get; set; }
+    DbSet<Ticket> Tickets { get; set; }
+    DbSet<TicketRoute> TicketRoutes { get; set; }
+    DbSet<Country> Countries { get; set; }
+    DbSet<Language> Languages { get; set; }
+    DbSet<Trip> Trips { get; set; }
+    DbSet<Waypoint> Waypoints { get; set; }
+    DbSet<WaypointFile> WaypointFiles { get; set; }
+
+    int SaveChanges();
+  }
```

Comment:
I think I have heard somewhere that each class is a separate file, each interface is a separate file, regardless of where it is implemented